.github/workflows/nightly-uds-core.yaml

```diff
@@ -16,6 +16,14 @@ concurrency:
 jobs:
   uds-core-test:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        existing-cluster: [with-cluster, without-cluster]
+        include:
+          - existing-cluster: without-cluster
+            create-cluster: 'false'
+          - existing-cluster: with-cluster
+            create-cluster: 'true'
```

Comment:
Open to suggestions on naming here, but basically wanted to convey that one matrix test operates by installing k3d and creating a cluster, while the other specifically does not create the cluster.

Reply:
Would it be clearer if this were just existing-cluster as a true/false and then pass the inverse to the action below? I'm not a big fan of the second matrix parameter here when it's really the same as the first just different name and truthy vs descriptive string.

Reply:
> forgot that the matrix value gets used in the job name when viewing it, so true/false would be a bit weird and unhelpful there.

Maybe this depends on how it's named? In this case, the _label_ is being used, not the value:

![with-cluster](https://github.com/user-attachments/assets/004bf9c4-6ba0-445b-bddd-3a4154c7ecc6)

But maybe I'm misunderstanding what you're saying here, I'm not fully versed in GitHub Actions these days

Reply:
ah ok re-reading this I believe you're referring to `existing-cluster: [with-cluster, without-cluster]`, I thought you were referring to the `include:` part.

Reply:
Added suggestions that I _think_ would work? Would also need to do the same on the release test of course, but wanted to add for your thoughts.

Reply:
@mjnagel I applied your suggestions, thanks! At first run I found the logic was backwards (`with-` did not create a cluster, but `without-` did) so I swapped it in e1c460c which I _believe_ is correct now. Watching CI run..